build/common.props

```diff
@@ -1,28 +1,32 @@
 <Project>
-
   <PropertyGroup>
+    <!-- we don't want to spawn any-long living compiler processes to avoid BenchmarkDotNet issues with "file in use" -->
```

Comment:
nit: Maybe this:
```xml
<!--
Avoid spawning any-long living compiler processes to avoid BenchmarkDotNet issues with "file in use",
and automation failing to cleanup the folders once the runs are over.
-->
```

Reply:
good point to mention the automation, I am going to update the comment